modules/api/src/main/scala/fi/spectrum/api/v1/services/HistoryApi.scala

```diff
@@ -37,6 +44,12 @@ object HistoryApi {
     e: ErgoAddressEncoder
   ) extends HistoryApi[F] {
 
+    def addressesHistory(paging: Paging): F[AddressesHistoryResponse] =
+      (for {
+        count <- history.countAllAddresses
+        result <- history.getAllAddresses(paging)
```

Comment:
map this result using 
`fi.spectrum.core.domain.address.formAddress(Redeemer.PublicKeyRedeemer(PubKey))` cause in db we keep not raw address but ergoTree